main/templates/main/hospital_filter.html

```diff
@@ -3,75 +3,76 @@
 {% load l10n %}
 
 {% block styles %}
-    <link rel="stylesheet" href="{% static "/css/leaflet.css" %}">
-    <link rel="stylesheet" href="{% static "/css/main.css" %}">
+<meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=no" />
+
+<link rel="stylesheet" href="https://unpkg.com/leaflet@1.6.0/dist/leaflet.css"
+      integrity="sha512-xwE/Az9zrjBIphAcBb3F6JVqxf46+CDLwfLMHloNu6KEQCAWi6HcDUbeOfBIptF7tcCzusKFjFw2yuvEpDL9wQ=="
+      crossorigin=""/>
+
+<link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/leaflet-easybutton@2/src/easy-button.css">
+
+<link rel="stylesheet" href="{% static "/css/map.css" %}">
 {% endblock styles %}
 
 {% block nav2 %}
 <nav class="flex bg-hospitalaid-blue-dark p-3 text-white">
-    <div class="flex-grow">
-      <h1 class="p-3 text-3xl font-bold">Aid requests</h1>
-    </div>
-    <div class="p-5 my-2 rounded-full">
-      <a href="{% url 'aidrequestfordonor_list' %}">
-        <img src="{% static "images/list-white.svg" %}" alt="list">
-      </a>
-    </div>
-    <div class="p-5 my-2 rounded-full bg-white">
-      <img src="{% static "images/map-blue.svg"  %}" alt="map">
+  <span class="flex-grow whitespace-no-wrap self-center text-3xl font-bold">Aid requests</span>
+  <a class="flex flex-shrink-0 self-center" href="{% url 'aidrequestfordonor_list' %}">
```

Comment:
Shouldn't be this section exactly like the section on the aidrequest_filter template?

Reply:
No. They are different.

hospital_filter related screen:
https://app.zeplin.io/project/5eb348cc15c3b04b8c301a69/screen/5eb349154c23207639bd7955

aidrequest_filter related screen:
https://app.zeplin.io/project/5eb348cc15c3b04b8c301a69/screen/5eb3491506a63c4b84e5fd9a